gslib/utils/copy_helper.py

```diff
@@ -3694,9 +3694,13 @@ def _WriteRowToManifestFile(self, url):
     # Aquire a lock to prevent multiple threads writing to the same file at
     # the same time. This would cause a garbled mess in the manifest file.
     with self.lock:
-      with open(self.manifest_path, 'a', 1) as f:  # 1 == line buffered
-        writer = csv.writer(f)
-        writer.writerow(data)
+      if IS_WINDOWS and six.PY3:
+        f = open(self.manifest_path, 'a', 1, newline='')
+      else:
+        f = open(self.manifest_path, 'a', 1)  # 1 == line buffered
```

Comment:
Our of curiosity, why do we specify line buffering specifically here instead of using [open's default buffering behavior](https://docs.python.org/3/library/functions.html#open)?

Reply:
I'm fine with line-buffering this... I'm assuming this is here so that we don't end up with a half-written line, if we encounter some weird error with one of the fields?

Reply:
I'm not completely sure why it was there, I only added the newline arg...